.changeset/mdx-content-component-types.md

```diff
@@ -0,0 +1,6 @@
+---
+'astro': patch
+'@astrojs/mdx': patch
+---
+
+Adds TypeScript support for the `components` prop on MDX `Content` component when using `await render()`. Developers now get proper intellisense and type checking when passing custom components to override default MDX element rendering.
```

Comment:
```suggestion
Adds TypeScript support for the `components` prop on MDX `Content` component when using `await render()`. Developers now get proper IntelliSense and type checking when passing custom components to override default MDX element rendering.
```
Looks great, but IntelliSense is a proper name and uses the above capitalization!